docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -73,15 +70,15 @@ SWIFT_PASSWORD=$(cat ~/tripleo-standalone-passwords.yaml | grep ' SwiftPassword:
 
 .Procedure
 
-. Make sure you are using the {OpenShift} namespace where you want the
-control plane deployed:
+. Ensure that you are using the {OpenShift} namespace where you want the
```

Comment:
This looks like it will expand to "Ensure that you are using the Red Hat OpenShift Container Platform namespace where..."
This isn't incorrect, but if there's a chance the customer won't be confused at this point to simply say "namespace", then it might be easier to read through.